actions/ql/src/Security/CWE-829/UntrustedCheckoutIssueCommentCritical.ql

```diff
@@ -6,7 +6,7 @@
  * @kind path-problem
  * @problem.severity warning
  * @precision high
- * @security-severity 7.5
+ * @security-severity 0.0
```

Comment:
I think we'll want to avoid this. We can discuss internally. Do we have known CVEs involving this problem, that we can use to compute the severity based on CVSS?